xcodeproj/internal/bazel_integration_files/calculate_output_groups.py

```diff
@@ -213,15 +214,33 @@ def _calculate_guid_labels_and_target_ids(base_objroot):
         compile_target_ids = {}
         for configuration in target_pif["buildConfigurations"]:
             config_build_target_ids = {"key": "BAZEL_TARGET_ID"}
-            config_compile_target_ids = {"key": "BAZEL_COMPILE_TARGET_ID"}
+            config_compile_target_ids = {"key": "BAZEL_COMPILE_TARGET_IDS"}
             for key, value in configuration["buildSettings"].items():
                 if key.startswith("BAZEL_TARGET_ID"):
+                    if value == "$(BAZEL_TARGET_ID)":
+                        target_ids = value
```

Comment:
Is it bad that this isn't a list?

Reply:
I thought the polymorphism was a bit weird too. But figured it'd be nice to explicitly tie into the same `$(BAZEL_TARGET_ID)` substitution as before but 🤷.

Reply:
If it works, we can keep it. Can you add some more comments?